### PR TITLE
fix(landing-page):guest shall see landing page

### PR DESCRIPTION
**Pull request by Tuyisenge2**
How should this be manually tested?
Clone the repo and cd into it
Checkout to fix-landing-page and run npm install
Run npm run dev to start the application locally
navigate to landing page as guest , dont login

### CODE REVIEW

File: src/pages/LandingProduct.tsx [hunk 1]
```diff
@@ -29,6 +30,7 @@ const LandingProduct = () => {
 	const [openModel, setOpenModel] = useState(false);
 	const [openFilter, setOpenFilter] = useState(false);
 	const dispatch = useAppDispatch();
+	//const [wishes, setWishes] = useState<DynamicData[]>([]);
```
Comment (GarrixA) [inline src/pages/LandingProduct.tsx]: Remove this comments

File: src/pages/LandingProduct.tsx [hunk 2]
```diff
@@ -53,6 +55,9 @@ const LandingProduct = () => {
 	useEffect(() => {
 		dispatch(search(searchInputs)).unwrap();
 	}, [dispatch, searchInputs]);
+	// useEffect(() => {
```
Comment (GarrixA) [inline src/pages/LandingProduct.tsx]: Also these

File: src/pages/LandingProduct.tsx
Comment (GarrixA) [inline src/pages/LandingProduct.tsx:1]: Screenshot 2024-07-23 at 18 43 11
Screenshot 2024-07-23 at 18 38 38


This page does not look well

**Review by GarrixA — CHANGES_REQUESTED** [review]

File: src/pages/LandingProduct.tsx
Comment (Tuyisenge2) [reply src/pages/LandingProduct.tsx:1]: @GarrixA  I managed to make it better now

**Review by GarrixA — APPROVED** [review]

**Review by YvetteNyibuka — APPROVED** [review]

**Review by leandreAlly — APPROVED** [review]

**Pull request merged**